migrations/versions/087d3afb4e8e_.py

```diff
@@ -1,31 +0,0 @@
-"""empty message
```

Comment:
Never delete migrations.  I can't roll this out to production.

Reply:
I'll copy the migrations over from master and then try to set up my local database with that